README.md

```diff
@@ -99,6 +99,8 @@ For protobuf-based codegen integrated with the Maven build system, you can use
 </build>
 ```
 
+For Eclipse IDE check https://github.com/trustin/os-maven-plugin#issues-with-eclipse-m2e-or-other-ides
```

Comment:
This is so tiny update, can you mention the link in any style,
as you can without new commit/PR

ref xolstice/protobuf-maven-plugin#16